Project2/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">  
+    <title>MovieCard</title>
+</head>
+<body style="background-color:#8a2be2"; >
+<div class="card"> 
+<figure>
+<a href="https://www.youtube.com/watch?v=DYYtuKyMtY8"><img class="thumbnail" src="images\SpiderManFFHTrailer-01.png" alt="Spider-Man Far From Home Trailer"></a>
+</figure>
+<div class="words">
+    <h1>Spider-Man:</h1>
+    <h2>Far From Home</h2>
+    <h3>Releasing July 5th, 2019</h3>
+    <a href="https://www.rottentomatoes.com/m/spider_man_far_from_home"><button class="button1">Ratings Details</button></a>
+    <p>Peter Parker's relaxing European vacation takes an unexpected turn when Nick Fury suddenly shows up in 
+        his hotel room. Parker soon finds himself donning the Spider-Man suit to help Fury stop the evil Mysterio 
+        from wreaking havoc across the continent.</p>
+    <p>Spider-Man: Far From Home is an upcoming American superhero film based on the Marvel Comics character 
+        Spider-Man, produced by Columbia Pictures and Marvel Studios, and distributed by Sony Pictures Releasing. 
+        It is set to be the sequel to Spider-Man: Homecoming (2017) and the twenty-third film in the Marvel 
+        Cinematic Universe (MCU). The film is directed by Jon Watts, written by Chris McKenna and Erik Sommers, 
+        and stars Tom Holland as Peter Parker / Spider-Man, alongside Samuel L. Jackson, Zendaya, Cobie Smulders, 
+        Jon Favreau, J. B. Smoove, Jacob Batalon, Martin Starr, Marisa Tomei, and Jake Gyllenhaal. It follows Parker 
+        and his friends as they go on a school trip to Europe.</p>
+    <p><a href="https://www.wired.com/story/spider-man-far-from-home-trailer/">more...</a></p>
+    <a href="http://www.landmarkreccinemas.com/mpaa"><button class="button1">PG-13</button></a>
+    <a href="https://www.fandango.com/spider-man-far-from-home-2019-216033/movie-times"><button class="button2">Buy Tickets</button></a>
```

Comment:
See previous note about either/or `a` and `button`:
```suggestion
    <a href="https://www.fandango.com/spider-man-far-from-home-2019-216033/movie-times" class="button2">Buy Tickets</a>
```

Note that you reference a `button2` class here, but there are no styles for it in your stylesheet.